SMTPServer/ThreadPool.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include<iostream>
+
+#include <winsock2.h>
+#include <ws2tcpip.h>
+
+#include <condition_variable>
+#include <mutex>
+#include <vector>
+#include <thread>
+#include <map>
+#include <chrono>
+#include <memory>
+#include <functional>
+
+#pragma comment(lib, "Ws2_32.lib")
```

Comment:
Here's again you add Ws2_32.lib to your project dependecies. It's alredy there, because you add it in MailSession.h (line 10).